prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -71,8 +91,8 @@ public async Task SendProvisionerLinkAsync(string[] recipients, EnrolmentCertifi
 
             string subject = "New Access Request";
             string emailBody = (string.IsNullOrEmpty(provisionerName))
-                ? this.GetClinicManagerEmailBody(token.Enrollee, token)
-                : this.GetVendorEmailBody(token.Enrollee, token, provisionerName);
+                ? await _razorConverterService.RenderViewToStringAsync("/Views/Emails/VendorEmail.cshtml", new EmailParams(token, provisionerName))
+                : await _razorConverterService.RenderViewToStringAsync("/Views/Emails/OfficeManagerEmail.cshtml", new EmailParams(token));
```

Comment:
How about
`string view = string.IsNullOrEmpty(provisionerName) ? "/Views/Emails/VendorEmail.cshtml" : "/Views/Emails/OfficeManagerEmail.cshtml";
string emailBody = _razorConverterService.RenderViewToStringAsync(view, new EmailParams(token, provisionerName);)`

Reply:
Also looks like the logic is reversed from the old code?